docs/starters.yml

```diff
@@ -1609,3 +1609,14 @@
     - Progressive Web app
     - Offline Support
     - RSS Feed
+- url: https://ueno.co/
+  repo: https://github.com/ueno-llc/ueno-gatsby-starter
+  description: Ueno's starter template for Gatsby
+  tags:
+    - TypeScript
+  features:
+    - GraphQL hybrid
+    - SEO friendly
+    - GSAP ready
+    - Nice devtools
+    - Decorators
```

Comment:
Are you using decorators?

Reply:
Yes, we have decorator support through our plugin: https://github.com/ueno-llc/gatsby-plugin-ueno/blob/4e9c14acce8dc3df0f76f29817dcaacfc7db38df/src/gatsby-node.js#L4